packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/ToggleVariants.jsx

```diff
@@ -1,25 +1,26 @@
 import React, { useState } from 'react';
 import StatusView from 'terra-status-view';
+import SingleSelectField from 'terra-form-select/lib/SingleSelectField';
 
 const ToggleVariants = () => {
   const [variant, setVariant] = useState('error');
 
-  const handleOnSelect = (event) => {
-    setVariant(event.target.value);
+  const handleOnSelect = (value) => {
+    setVariant(value);
   };
 
   return (
     <div>
       <StatusView variant={variant} />
-      <fieldset>
-        <legend>Change Variant</legend>
-        <select aria-label="changeVariant" id="statusViewVariant" name="statusViewVariant" value={variant} onChange={handleOnSelect}>
-          <option value="error">Error</option>
-          <option value="no-data">No Data</option>
-          <option value="no-matching-results">No Matching Results</option>
-          <option value="not-authorized">Not Authorized</option>
-        </select>
-      </fieldset>
+      <SingleSelectField label="Change Variant" onChange={handleOnSelect} value={variant} maxWidth="300px" selectId="change-variant-field" placeholder="Select a Variant">
+        <SingleSelectField.Option value="error" display="Error" />
+        <SingleSelectField.Option value="no-data" display="No Results" />
+        <SingleSelectField.Option value="no-matching-results" display="No Matching Result" />
+        <SingleSelectField.Option value="not-authorized" display="Not Authorized" />
+      </SingleSelectField>
+      <p>
+        ** Here the Select field shows the list of available status view variants. One can select any of the available options to check. For example a simple search screen can use the No-results variant to show that the entered string is not available while fetching a result from the database.
```

Comment:
Since this seems to be giving instructions for using the example rather than how to use the Status View component, can we change to something like:

```suggestion
        Select a status view variant from the dropdown above.
```

Reply:
Changes made. Thanks